src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -76,43 +82,53 @@ end
 
 # this method is used when the limiter is constructed as for shock-capturing volume integrals
 function SubcellLimiterIDP(equations::AbstractEquations, basis;
-                           local_minmax_variables_cons = String[],
+                           local_twosided_variables_cons = String[],
                            positivity_variables_cons = String[],
                            positivity_variables_nonlinear = [],
                            positivity_correction_factor = 0.1,
-                           spec_entropy = false,
-                           math_entropy = false,
+                           local_onesided_variables_nonlinear = [],
                            max_iterations_newton = 10,
                            newton_tolerances = (1.0e-12, 1.0e-14),
                            gamma_constant_newton = 2 * ndims(equations))
-    local_minmax = (length(local_minmax_variables_cons) > 0)
+    local_twosided = (length(local_twosided_variables_cons) > 0)
+    local_onesided = (length(local_onesided_variables_nonlinear) > 0)
     positivity = (length(positivity_variables_cons) +
                   length(positivity_variables_nonlinear) > 0)
-    if math_entropy && spec_entropy
-        error("Only one of the two can be selected: math_entropy/spec_entropy")
+
+    # When passing `min` or `max` in the elixir the specific function of Base is used.
+    # To speed up the simulation, we replace it with `Trixi.min` and `Trixi.max` respectively.
+    local_onesided_variables_nonlinear_ = Tuple{Function, Function}[]
+    for (variable, operator) in local_onesided_variables_nonlinear
+        if operator === Base.max
+            push!(local_onesided_variables_nonlinear_, tuple(variable, max))
+        elseif operator === Base.min
+            push!(local_onesided_variables_nonlinear_, tuple(variable, min))
```

Comment:
Maybe allow the case of `Trixi.min` and `Trixi.max`?
```suggestion
            push!(local_onesided_variables_nonlinear_, tuple(variable, min))
        elseif operator === min || operator === max
            push!(local_onesided_variables_nonlinear_, tuple(variable, operator))
```